gpcontrib/arenadata_toolkit/arenadata_toolkit--1.3--1.4.sql

```diff
@@ -0,0 +1,379 @@
+/* gpcontrib/arenadata_toolkit/arenadata_toolkit--1.3--1.4.sql */
+
+DO $$
+BEGIN
+	IF (SELECT c.oid IS NOT NULL
+		   AND a.attrelid IS NULL
+		FROM pg_class c
+		JOIN pg_namespace n ON n.oid = c.relnamespace
+		LEFT JOIN pg_attribute a ON a.attrelid = c.oid AND
+		                            a.attname = 'tablespace_location'
+		WHERE c.relname = 'db_files_current' AND
+		      n.nspname = 'arenadata_toolkit')
+	THEN
+		ALTER TABLE arenadata_toolkit.db_files_current
+		ADD COLUMN tablespace_location TEXT;
+	END IF;
+END $$;
+
+DO $$
+BEGIN
+	IF EXISTS (SELECT 1
+			   FROM pg_tables
+			   WHERE schemaname LIKE 'pg_temp%' AND
```

Comment:
Why did you change equality to start checking?

Reply:
Why are you removing the db_files_current table from the pg_temp schema if it is in the pg_temp_25 schema?

Reply:
```sql
postgres=# CREATE TEMPORARY TABLE IF NOT EXISTS pg_temp.db_files_current
        (
                oid BIGINT,
                table_name TEXT,
                table_schema TEXT,
                type CHAR(1),
                storage CHAR(1),
                table_parent_table TEXT,
                table_parent_schema TEXT,
                table_database TEXT,
                table_tablespace TEXT,
                "content" INTEGER,
                segment_preferred_role CHAR(1),
                hostname TEXT,
                address TEXT,
                file TEXT,
                modifiedtime TIMESTAMP WITHOUT TIME ZONE,
                file_size BIGINT,
                tablespace_location TEXT
        )
        DISTRIBUTED RANDOMLY;
CREATE TABLE
postgres=# select schemaname, tablename from pg_tables where schemaname LIKE 'pg_temp%' AND tablename = 'db_files_current';
 schemaname |    tablename     
------------+------------------
 pg_temp_25 | db_files_current
(1 row)

postgres=# DROP TABLE pg_temp.db_files_current;
DROP TABLE
postgres=# select schemaname, tablename from pg_tables where schemaname LIKE 'pg_temp%' AND tablename = 'db_files_current';
 schemaname | tablename 
------------+-----------
(0 rows)
```